### PR TITLE
inspect quiet should be restored to previous value

### LINKED ISSUE #232: Mixing sregistry and spython problems
**Describe the bug**

Hi @vsoch!

In one of my Python scripts I am creating Singularity images in a for loop, with each image being built on top of the previous one.

I am running into trouble when building the second image, as I noticed that `sudo singularity build` is then invoked with the `--quiet` parameter, which is an issue for me since my sudo privileges do not allow this. I was wondering why this happened because I nowhere asked for quiet mode and figured out that this happens after the first image is created with `spython.main.Client.build`, when I add the image to my local database with my sregistry client, which appears to set the spython client quiet attribute to True, when running inspect:

https://github.com/singularityhub/sregistry-cli/blob/54c0c3ed18a4bae005863ae706e1b07b37a6297f/sregistry/main/base/inspect.py#L58-L63

My suggestion is to fix this in sregistry by saving the current quiet attribute value before calling inspect, and restoring it afterwards.

**To Reproduce**

A bit complicated to reduce to a couple of lines  so I recommend to have a look at https://github.com/DiamondLightSource/Savu/blob/c3677cf69a3be0247d517aa76907bae136d100cc/singularity-recipes/mass-rebuild.py, which is where I am using this.

**Version of Singularity and Singularity Registry Client**

Singularity version: 2.6.1
Singularity Registry version: 0.2.25
Singularity CLI version: 0.0.73

I hope this is clear, let me know if you need to know anything else!

### PR DESCRIPTION
**Pull request by vsoch**
This will close #232 

Signed-off-by: Vanessa Sochat <vsochat@stanford.edu>

### CODE REVIEW

File: sregistry/main/base/inspect.py [hunk 1]
```diff
@@ -54,10 +54,14 @@ def get_metadata(self, image_file, names=None):
...
         from spython.main import Client as Singularity
 
+        # Store the original quiet setting
+        is_quiet = Singularity.quiet
+
         # We try and inspect, but not required (wont work within Docker)
         try:
             Singularity.quiet = True
             updates = Singularity.inspect(image=image_file)
+            Singularity.quiet = is_quiet
```
Comment (tschoonj) [inline sregistry/main/base/inspect.py]: Would it make sense to restore the attribute value in a finally clause?

Comment (vsoch) [reply sregistry/main/base/inspect.py]: Yes, good call. If the update fails we never hit the last statement (but we'll hit the first!). Change coming right up!

**Comment by vsoch** [comment]
Ready for merge @tschoonj ?

**Comment by vsoch** [comment]
Released https://pypi.org/project/sregistry/0.2.25/

**Pull request merged**